src/include/cursor.h

```diff
@@ -228,10 +228,11 @@ struct __wt_cursor_btree {
 #define	WT_CBT_ITERATE_NEXT	0x004u	/* Next iteration configuration */
 #define	WT_CBT_ITERATE_PREV	0x008u	/* Prev iteration configuration */
 #define	WT_CBT_NO_TXN   	0x010u	/* Non-txn cursor (e.g. a checkpoint) */
-#define	WT_CBT_RETRY_NEXT	0x020u	/* Next, resulted in prepare conflict */
-#define	WT_CBT_RETRY_PREV	0x040u	/* Prev, resulted in prepare conflict */
-#define	WT_CBT_SEARCH_SMALLEST	0x080u	/* Row-store: small-key insert list */
-#define	WT_CBT_VAR_ONPAGE_MATCH	0x100u	/* Var-store: on-page recno match */
+#define	WT_CBT_READ_WONT_NEED	0x020u	/* Page in with WT_READ_WONT_NEED */
```

Comment:
I'm not happy with the comment here, but I couldn't make it fit in under 80 characters. I wasn't sure how to style the definition if it exceeded 80 characters.